docs/release_notes.md

```diff
@@ -5,6 +5,125 @@ hide:
+- [[PR #11883]](https://github.com/kubevirt/kubevirt/issues/11883) [orelmisan] Restart of a VM is required when the CPU socket count is reduced
+- [[PR #11655]](https://github.com/kubevirt/kubevirt/issues/11655) [acardace] Allow to hotplug vcpus for VMs with CPU requests and/or limits set
+
+### SIG-storage
+- [[PR #11095]](https://github.com/kubevirt/kubevirt/issues/11095) [ShellyKa13] Expose volumesnapshot error in vmsnapshot object
+- [[PR #11312]](https://github.com/kubevirt/kubevirt/issues/11312) [alromeros] Improve handling of export resources in virtctl vmexport
+- [[PR #11770]](https://github.com/kubevirt/kubevirt/issues/11770) [alicefr] Fix the live updates for volumes and disks
+- [[PR #11957]](https://github.com/kubevirt/kubevirt/issues/11957) [mhenriks] snapshot: Ignore unfreeze error if VMSnapshot deleting
+
+### SIG-network
+- [[PR #11653]](https://github.com/kubevirt/kubevirt/issues/11653) [EdDev] Build the `passt`custom CNI binary statically, for the `passt` network binding plugin.
+- [[PR #11678]](https://github.com/kubevirt/kubevirt/issues/11678) [Vicente-Cheng] Improve the handling of ordinal pod interface name for upgrade
+- [[PR #11618]](https://github.com/kubevirt/kubevirt/issues/11618) [AlonaKaplan] Extend network binding plugin to support device-info DownwardAPI.
+- [[PR #11256]](https://github.com/kubevirt/kubevirt/issues/11256) [andreabolognani] This version of KubeVirt includes upgraded virtualization technology based on libvirt 10.0.0 and QEMU 8.2.0.
+- [[PR #11788]](https://github.com/kubevirt/kubevirt/issues/11788) [ormergi] The network-info annotation is now used for mapping between SR-IOV network and the underlying device PCI address
+- [[PR #11659]](https://github.com/kubevirt/kubevirt/issues/11659) [iholder101] Require scheduling infra components onto control-plane nodes
+- [[PR #12079]](https://github.com/kubevirt/kubevirt/issues/12079) [EdDev] Network hotplug feature is declared as Beta.
+
+### SIG-infra
+- [[PR #11648]](https://github.com/kubevirt/kubevirt/issues/11648) [kubevirt-bot] Updated common-instancetypes bundles to v1.0.0
+- [[PR #11962]](https://github.com/kubevirt/kubevirt/issues/11962) [lyarwood] `VirtualMachines` referencing an instance type are now allowed when the `LiveUpdate` feature is enabled and will trigger the `RestartRequired` condition if the reference within the `VirtualMachine` is changed.
+- [[PR #11455]](https://github.com/kubevirt/kubevirt/issues/11455) [lyarwood] `LiveUpdates`  of VMs using instance types are now supported with the same caveats as when making changes to a vanilla VM.
+- [[PR #11681]](https://github.com/kubevirt/kubevirt/issues/11681) [lyarwood] The `CommonInstancetypesDeployment` feature and gate are retrospectively moved to Beta from the 1.2.0 release.
+- [[PR #12171]](https://github.com/kubevirt/kubevirt/issues/12171) [kubevirt-bot] `PreferredDiskDedicatedIoThread` is now only applied to `virtio` disk devices
```

Comment:
This was my bugfix backported after the branch was cut but it's not a massive issue.